hack/managed-kyma-migration/readme.md

```diff
@@ -1,112 +1,119 @@
+
+1. Create a Dev SAP BTP, Kyma runtime cluster.
 2. Add the Keda module to the Kyma CR.
    ```yaml
    spec:
      modules:
      - name: keda
    ```
 3. Execute the migration.
-4. Verify that `istio-manager` is installed and the Istio CR's status is `Ready`.
-5. Verify that the Kyma reconciler does not reconcile the Istio component anymore.
-   ```shell
-   # List reconciliations to get the last scheduling ID
-   kcp rc -c {SHOOT}
-   # List reconciled components
-   kcp rc info -i {SCHEDULING ID}
-   ```   
+4. Verify that `istio-controller-manager` is installed and the Istio CR's status is `Ready`.
 
 ## Module's rollout and migration
 
 ### Preparations
-1. Executing `kcp taskrun` requires the path to the kubeconfig file of the corresponding Gardener project and permissions to list/get shoots.
-2. Set `metadata.name` in the module template to `istio-regular`.
-3. Set `spec.channel` in the module template to `regular`.
+
+Executing `kcp taskrun` requires the path to the kubeconfig file of the corresponding Gardener project and permissions to list/get shoots.
 
 ### Dev
-1. Apply the module template to Dev Control Plane.
-2. `kcp login` to Dev and run the migration script on all SKRs. To do that, you can use the following command:
-   ```shell 
-   kcp taskrun --gardener-kubeconfig {PATH TO GARDENER PROJECT KUBECONFIG} --gardener-namespace kyma-dev -t all -- ./migrate.sh
+
+#### Migration procedure
+
+1. Push module to `experimental` channel in `kyma/module-manifests `repository (PR #162).
+2. Push the kustomization change for `experimental` in `kyma/kyma-modules` repository (PR #400)
+3. Verify that the ModuleTemplate is present in the `kyma/kyma-modules` internal repository.
+4. Apply the ModuleTemplate for `fast` and `regular` channels to Dev Control Plane.
+5. Verify that the ModuleTemplate in the `experimental`, `fast` and `regular` channels is available on SAP BTP, Kyma runtime clusters of the Dev environment.
+6. Merge PR (#4624) in `kyma/management-plane-config` responsible for disabling Istio reconciliation and setting Istio as a default module on dev. Depending on the outcome adjust this those actions to one PR for the stage and prod.
+7. Use `kcp login` to log in to Dev and run the migration script on all SAP BTP, Kyma runtime clusters. To do that, you can use the following command:
+   ```shell
+   kcp taskrun --gardener-kubeconfig {PATH TO GARDENER PROJECT KUBECONFIG} -t all -- ./managed-kyma-migration.sh
    ```
-3. Verify that the migration worked as expected by checking the status of Istio manifests on Control Plane.
+8. Verify that the migration worked as expected by checking the status of Istio manifests on Control Plane.
    ```shell
    kubectl get manifests -n kcp-system -o custom-columns=NAME:metadata.name,STATE:status.state | grep istio
    ```
+9. If there are APIGateway CR is in the warning state enable Istio Module in the Kyma CR (you can check it easily on the new monitoring dashboard).
```

Comment:
```suggestion
9. Verify that no APIGateway module is in Warning state. This can be done by using the [API-Gateway Monitoring Dashboard](https://plutono.cp.dev.kyma.cloud.sap/d/6meO06VSk/modules-api-gateway?orgId=1).
10. If there are APIGateway CR is in the warning state, the cluster might have been created without Istio Module. In this case it needs to be enabled in the Kyma CR manually.
```